app/src/main/kotlin/io/sweers/catchup/ui/fragments/SmmryFragment.kt

```diff
@@ -178,77 +163,70 @@ class SmmryFragment : InjectableBaseFragment() {
     } else {
       loadingView.hide()
     }
-    Maybe.concatArray(tryRequestFromStorage(), getRequestSingle().toMaybe())
-        .firstOrError()
-        .subscribeOn(Schedulers.io())
-        .observeOn(AndroidSchedulers.mainThread())
-        .autoDisposable(this)
-        .subscribe({ smmryResponse ->
-          alreadyLoaded = true
-          loadingView.hide(true)
-          when (smmryResponse) {
-            is Success -> {
-              showSummary(smmryResponse)
-            }
-            else -> {
-              val message = when (smmryResponse) {
-                is InternalError -> "Smmry internal error - ${smmryResponse.message}"
-                is IncorrectVariables -> "Smmry invalid input - ${smmryResponse.message}"
-                is ApiRejection -> "Smmry API error - ${smmryResponse.message}"
-                is SummarizationError -> "Smmry summarization error - ${smmryResponse.message}"
-                UnknownErrorCode -> "Unknown error :("
-                else -> TODO("Give me sealed whens!")
-              }
-              summary.text = message
-            }
-          }
-        }, { error ->
-          loadingView.hide(true)
-          summary.setText(R.string.unknown_issue)
-          if (error is IOException) {
-            w(error) { "Unknown error in smmry load" }
-          } else {
-            e(error) { "Unknown error in smmry load" }
+    lifecycleCoroutineScope().launch {
```

Comment:
you probably want to assign `lifecycleCoroutineScope()` to a private val in the fragment itself

Reply:
Cleaned up in d349407